en/docs/guides/login/logging-into-simplesamlphp-using-is.md

```diff
@@ -0,0 +1,209 @@
+# Logging in to SimpleSAMLphp using WSO2 Identity Server
+
+This section explains how to configure the WSO2 Identity Server with
+SimpleSAMLphp as a service provider. Initially, it is necessary to setup
```

Comment:
Shall we use the same kind of introductory sentence as the other pages?

"This page guides you through using WSO2 Identity Server to log in to SimpleSAMLphp"

Reply:
setup --> set up